idea/idea-frontend-fir/src/org/jetbrains/kotlin/idea/frontend/api/fir/symbols/KtFirSyntheticJavaPropertySymbol.kt

```diff
@@ -83,6 +83,7 @@ internal class KtFirSyntheticJavaPropertySymbol(
     override val javaSetterName: Name? get() = firRef.withFir { it.setter?.delegate?.name }
 
     override val isOverride: Boolean get() = firRef.withFir { it.isOverride }
+    override val isStatic: Boolean get() = false
```

Comment:
I am not sure why do you use `= false` here. Seems like synthetic property can be static (when it is just a plain old java field)

Reply:
Good point! I missed that!